### PR TITLE
Add Cluster Operator upgrade activity

### PR DESCRIPTION
**Pull request by rawagner**
co_upgrade

Update/Upgrade wording fix for dashboards and cluster operators page will follow

@andybraren

### CODE REVIEW

**Comment by rawagner** [comment]
Or something like this ? @andybraren 
upgrade_alt

**Comment by andybraren** [comment]
My thought would be to align with the same format for all other resource-related activities, which I believe is:

- The time the upgrade started (if available, no problem if not)
- The same Status label that's shown in the OLM/CO List views (I'm not quite sure where to find this, so I'll assume it's "Upgrading" for now)
- A Status Popover if upgrades could take a long time and we have more information to share with the user (possibly "Progressing: not all deployment replicas are ready" in this case, but that message doesn't seem super useful to me unless it changes as it progresses)
- The resource badge and name of the OLM/CO operator below the status

Does this PR replace #3575 by the way?

concept-operator-activity

**Comment by rawagner** [comment]
Thanks, i will update the code accodingly
> My thought would be to align with the same format for all other resource-related activities, which I believe is:
> 
> * The time the upgrade started (if available, no problem if not)
> * The same Status label that's shown in the OLM/CO List views (I'm not quite sure where to find this, so I'll assume it's "Upgrading" for now)
> * A Status Popover if upgrades could take a long time and we have more information to share with the user (possibly "Progressing: not all deployment replicas are ready" in this case, but that message doesn't seem super useful to me unless it changes as it progresses)
Yea, im not sure if we can get other messages, i will try to find out and possiblity just remove it

> * The resource badge and name of the OLM/CO operator below the status
> 
> Does this PR replace #3575 by the way?
it does not. This is for Cluster operators and #3575 is for CVS (OLM operators). I will update #3575 to look the same too.

> 
> concept-operator-activity

**Comment by rawagner** [comment]
PR updated
co_upgrade_new

File: frontend/packages/console-app/src/components/dashboards-page/ClusterOperatorUpgradeActivity.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import './activity.scss';
+
+const ClusterOperatorUpgradeActivity: React.FC<ClusterOperatorUpgradeActivityProps> = ({
+  resource,
+}) => (
+  <>
+    <ActivityItem>Upgrading</ActivityItem>
+    <ResourceLink
+      kind={referenceForModel(ClusterOperatorModel)}
+      name={resource.metadata.name}
```
Comment (jtomasek) [inline frontend/packages/console-app/src/components/dashboards-page/ClusterOperatorUpgradeActivity.tsx]:
```suggestion
      name={getName(resource)}
```

Comment (rawagner) [reply frontend/packages/console-app/src/components/dashboards-page/ClusterOperatorUpgradeActivity.tsx]: fixed

**Pull request merged**